geomstats/geometry/invariant_metric.py

```diff
@@ -595,7 +595,7 @@ def lie_acceleration(point, vector):
                             step=step, **kwargs)
         return flow[-1]
 
-    def log(self, point, base_point, n_steps=15, step='k4',
+    def log(self, point, base_point, n_steps=15, step='rk4',
```

Comment:
From the docstring it seems that step should be in group_rk2, group_rk4, and that the default is group_rk4?

Maybe add a check:
```
if step not in ['group_rk2', 'group_rk4']:
....
```
?

Reply:
I think the docstring is not up to date, I harmonized all the steps so that there is no longer group specific ones

Reply:
And there is `check_parameter_accepted_values(step, 'step', STEP_FUNCTIONS)` in integrator